### ISSUE TITLE
Hamburger button missing label and attributes

### ISSUE THREAD

**Issue by jamcgrath** [owddm/owddm.github.io#196]
Hamburger menu button has the following issues

- needs to have the href attribute removed
- missing `type="button"`
- missing text label suggest either put `aria-label` or `<span class="sr-only" >` in the button

**Comment by kinxiel** [comment]
Thanks for spotting this yet again. Addressed in the latest commit.
